itea.html

```diff
@@ -52,7 +52,7 @@ <h1>IS Tea Round Cheat Sheet</h1>
 			<tr><td><strong>Dave</strong></td><td> HPA, else some lager like San Miguel. Estrella in the Robin Hood. </td></tr>
 			<tr><td><strong>Luke</strong></td><td> Bitter. If there's one with an amusing name, go for that.</td></tr>
 			<tr><td><strong>Gary</strong></td><td> Usually Guinness. After playing badders, a refreshing pint of Lager.</td></tr>
-			<tr><td><strong>James</strong></td><td> <del>Pepsi</del> Coca Cola</td></tr>
+			<tr><td><strong>Chris</strong></td><td>Usually Guinness. Also enjoys</td></tr>
```

Comment:
You've got Roy and Chris in the wrong order in this section

Reply:
new commit.